packages/perseus/src/widgets/interactive-graphs/graphs/types.ts

```diff
@@ -0,0 +1,10 @@
+import type {CollinearTuple} from "../../../perseus-types";
+import type {vec} from "mafs";
+
+export interface InteractiveLineProps {
+    collinearPair: Readonly<CollinearTuple>;
```

Comment:
Can this be a tuple like other places you updated things?

```suggestion
    collinearPair: [CollinearTuple, CollinearTuple];
```

Reply:
Yes, it should be! This was changed very early in the process. Good catch!

Reply:
Oh-- this isn't a ReadonlyArray, it's just Readonly!